setup.py

```diff
@@ -27,7 +27,7 @@
     if sys.platform in ["win32"]:
         extensions.append(Extension("simulator", sources=sources, include_dirs=[os.path.join(path, "")], extra_compile_args=["/d2FH4-"]))
     elif sys.platform.startswith("darwin"):
-        extensions.append(Extension("simulator", sources=sources, include_dirs=[os.path.join(path, "")], extra_compile_args=["-Xpreprocessor"]))
+        extensions.append(Extension("simulator", language="c", sources=sources, include_dirs=[os.path.join(path, "")], extra_compile_args=["-Xpreprocessor"]))
```

Comment:
Could you clarify why this change is needed?

Reply:
The option language is not required as one would ned to append compiler flag for clang anyway at the end when compiling with 'c'.